instrumentation-core/spring/spring-web-3.1/spring-web-3.1.gradle

```
@@ -1,25 +1,9 @@
-group = 'io.opentelemetry.instrumentation'
+ext {
+  javaSubPackage = ''
+}
```

maybe something more explicit which disables the `shadowJar` task in `instrumentation-library.gradle`, e.g.

we could also have lack of `javaSubPackage` disable the `shadowJar` task, but don't want people to accidentally forget about shading these dependencies for auto-instrumentation, so think maybe this is safer

```suggestion
  noShadow = true
```

---

oh sorry, i accidentally removed the surrounding `ext { ... }`

---

I run into this error when I don't define javaSubPackage:

```
A problem occurred evaluating script.
> Could not get unknown property 'javaSubPackage'
```
Is there a way to disable the shadowJar configuration in instrumentation-library.gradle so that I don't run into this compile time error? A work around would be to just set javaSubPackage to an empty string but that doesn't seem ideal.

---

there may be a better way, but wrapping parts of `instrumentation-library.gradle` in conditionals seems to do the trick:

```
if (!ext.properties.noShadow) {
  apply plugin: 'com.github.johnrengelman.shadow'
}
```

if (!ext.properties.noShadow) {
  shadowJar {
    archiveClassifier = 'agent'

    configurations = []

    relocate "io.opentelemetry.instrumentation.${javaSubPackage}", "io.opentelemetry.auto.instrumentation.${javaSubPackage}.shaded"
  }
}

---

Awesome thanks!